framework/db/Command.php

```diff
@@ -868,9 +870,12 @@ public function execute()
      */
     protected function queryInternal($method, $fetchMode = null)
     {
-        $rawSql = $this->getRawSql();
-
-        Yii::info($rawSql, 'yii\db\Command::query');
+        $log = $this->db->enableQueryLog;
+        if ($log) {
+            $rawSql = $this->getRawSql();
+            Yii::info($rawSql, 'yii\db\Command::query');
+            $token = $rawSql;
+        }
```

Comment:
This block duplicates the one from `execute()`, which is one more argument for https://github.com/yiisoft/yii2/pull/14056/files#r113421830